helm/hpcc/values.yaml

```diff
@@ -82,8 +82,12 @@ global:
       #labels:
       #  mylabel: "4"
       type: LoadBalancer
-      
-      
+    # Ingress Network Polices will be created for following esp pods. The Network Policies will only take effect
+    # when a Kubernetes supported network polocy provider is ininstalled.
+    exposedServices: ["eclwatch", "eclqueries", "esdl-sandbox", "sql2ecl"]
+    # The following settings can be overwritted by ingress rules defined esp services/ingress
+    ingress:
```

Comment:
ingress section should be within each of the user-defined sections above. Any "visibility" that has type "loadbalancer" probably wants to also define its own ingress settings, which are specific to that visibility rule.

Reply:
I thought we wanted a global default ingress rules for the services which has type "loadbalancer"